samples/js-shopping-cart/js-shopping-cart.yaml

```diff
@@ -3,14 +3,16 @@ apiVersion: cloudstate.io/v1alpha1
 kind: StatefulService
 metadata:
   name: shopping-cart
+  labels:
+    app: shopping-cart
 spec:
-
   # Datastore configuration
-  storeConfig:
-    statefulStore:
-        # Name of a deployed Datastore to use.
-        name: inmemory
-
+  datastore:
+    # Name of a deployed Datastore to use.
+    name: cassandra
+    # Datastore specific configuration
+    config:
+      # Since the datastore is a Cassandra datastore, we need to specify the keyspace to use.
+      keyspace: shoppingcart
```

Comment:
This was also changed for the new operator:

https://github.com/cloudstateio/cloudstate/pull/425/files#diff-01dc38e97c1ec4b24976edc69721cba2L9-L18